tests/src/test/scala/it/unibo/scafi/test/functional/stdlib/BlockT.scala

```diff
@@ -0,0 +1,117 @@
+package it.unibo.scafi.test.functional.stdlib
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import it.unibo.scafi.test.functional.ScafiAssertions.assertNetworkValues
+import it.unibo.scafi.test.functional.ScafiTestUtils
+import it.unibo.utils.StatisticsUtils._
+import org.scalatest._
+
+
+/*
+Not yet tested:
+- cyclicTimerWithDecay
+- clock
+- impulsesEvery
+ */
+class BlockT extends FlatSpec{
```

Comment:
`TestBlockT` is arguably a better name for the class and file

Reply:
I agree